### PR TITLE
Fix issue 4

### PR DESCRIPTION
**Pull request by RobertGauld**
Two commits:
1) Fix the actual problem (undefined method when using Rails 4.1)
2) Allow travis to run tests against both the current bundle and one containing rails 4.1.0.rc1

### CODE REVIEW

File: gemfiles/rails4.1.rc1 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+source "https://rubygems.org"
+
+gem 'activerecord', '4.1.0.rc1'
+gem 'bundler'
+gem 'coveralls'
+gem 'date_time_attribute', :path=>'../'
```
Comment (einzige) [inline gemfiles/rails4.1.rc1:6]: will it work without this line?

Comment (einzige) [inline gemfiles/rails4.1.rc1:6]: yup, https://travis-ci.org/einzige/date_time_attribute/builds/20042065

**Comment by einzige** [comment]
@robertgauld thank you very much man, appreciate it

bundled 0.0.7

**Comment by RobertGauld** [comment]
That's OK bud, it was a simple problem which gave me a chance to learn pull
requests by doing.

**Pull request merged**